examples/ibm-db2/README.md

```diff
@@ -0,0 +1,49 @@
+# This example shows how to create an IBM Cloud Db2 SaaS and configure connectivity from a VSI
+
+This sample configuration to provision an IBM Db2 SaaS instance. 
+
+## Costs
+
+This sample uses chargable services and **will** incur costs for the time the services are deployed. Execution of `terraform destroy` will result in deletion of all resources including the Db2 SaaS service instance. Billing for Db2 SaaS will terminate on the hour. 
+
+
+## Dependencies
+
+- User has IAM security rights to create and configure an IBM Cloud Db2 SaaS Instance.
```

Comment:
Change to "User has IAM permissions to create and configure an IBM Db2 SaaS for IBM Cloud Instance in the resource group specified."